src/main/kotlin/com/mparticle/kits/UrbanAirshipConfiguration.kt

```diff
@@ -67,6 +68,7 @@ class UrbanAirshipConfiguration(settings: Map<String, String>) {
         private const val KEY_APP_KEY = "applicationKey"
         private const val KEY_APP_SECRET = "applicationSecret"
         private const val KEY_DOMAIN = "domain"
+        private const val KEY_INITIAL_CONFIG_URL = "initialConfigUrl"
```

Comment:
Configuration setting key will be added to align with UI description.

```suggestion
        private const val KEY_CUSTOM_DOMAIN_PROXY_URL = "customDomainProxyUrl"
```